peps/pep-0735.rst

```diff
@@ -258,10 +245,6 @@ as defined in :pep:`508`.
 Tables in requirement lists must be valid Dependency Object Specifiers,
 defined below.
 
-The ``project`` table in ``pyproject.toml`` is modified such that
```

Comment:
On the line above this one, I suggest removing "defined below" as the Dependency Object Specifiers section follows this sentence.